<ui version="4.0" >
 <class>AnimateWidget</class>
 <widget class="QWidget" name="AnimateWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>92</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="start_button" >
       <property name="toolTip" >
        <string>Go to start</string>
       </property>
       <property name="statusTip" >
        <string>Backs until the first animation frame</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/default.qrc" >:/icons/default/icons/control_start.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Shift+Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reverse_button" >
       <property name="toolTip" >
        <string>Reverse</string>
       </property>
       <property name="statusTip" >
        <string>Animate reverse</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/default.qrc" >:/icons/default/icons/control_rewind.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause_button" >
       <property name="toolTip" >
        <string>Pause</string>
       </property>
       <property name="statusTip" >
        <string>Pauses the current animation</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/default.qrc" >:/icons/default/icons/control_pause.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Shift+Space</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forward_button" >
       <property name="toolTip" >
        <string>Forward</string>
       </property>
       <property name="statusTip" >
        <string>Animate forward</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/default.qrc" >:/icons/default/icons/control_fastforward.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="end_button" >
       <property name="toolTip" >
        <string>Go to end</string>
       </property>
       <property name="statusTip" >
        <string>Forwards until the last animation frame</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/default.qrc" >:/icons/default/icons/control_end.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Shift+End</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="repeat_button" >
       <property name="toolTip" >
        <string>Repeat</string>
       </property>
       <property name="statusTip" >
        <string>Enable/disable looping of animation</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../resource/default.qrc" >:/icons/default/icons/control_repeat.png</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>S&amp;peed</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy" >
        <cstring>speed_box</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="speed_box" >
       <property name="toolTip" >
        <string>Speed</string>
       </property>
       <property name="statusTip" >
        <string>Sets the speed of the animation</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/default.qrc" />
 </resources>
 <connections/>
</ui>
